Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -507,6 +508,28 @@ try {
         -appBuild $appBuild -appRevision $appRevision `
         -uninstallRemovedApps
 
+    # If any apps were downloaded as part of incremental builds in a pr, we should remove them again after the build
+    try {
+        if ($ENV:GITHUB_EVENT_NAME -like 'pull_request*' -and $downloadedAppsByType) {
+            $downloadedAppsByType | ForEach-Object {
+                if ($_.downloadedApps) {
+                    $mask = $_.mask
+                    $thisArtifactFolder = Join-Path $buildArtifactFolder $mask
+                    Write-Host "Removing downloaded apps from $thisArtifactFolder"
+                    foreach($downloadedApp in $_.downloadedApps) {
+                        $thisApp = Join-Path $thisArtifactFolder $downloadedApp
+                        Write-Host "Removing: $thisApp"
+                        if (Test-Path $thisApp) {
+                            Remove-Item $thisApp
+                        }
+                    }
+                }
+            }
+        }
+    } catch {
```

Comment:
Please move the try/catch to the Remove-Item and explain which app couldn't be deleted. This will help in troubleshooting scenarios.